client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-gallery-panel/incident-gallery-panel-mobile/incident-gallery-panel-mobile.component.ts

```diff
@@ -173,5 +176,25 @@ loadMoreVideos(e: HTMLElement) {
     } 
   }
 
+  setPrimaryToTop: any = (collection: any) => {
+    let itemToBeSpliced = null;
+    let index = null;
+    for (let item of collection){
+      if (item?.primary === 'true'){
+          itemToBeSpliced = item
+          index = collection.indexOf(item)
+       }
+    } console.log("itemToBeSpliced = " + itemToBeSpliced)
```

Comment:
We can get rid of the console statements.

Reply:
Resolved this now.